cusub_cortex/state_machine/src/tasks/droppers_det.py

```diff
@@ -208,8 +209,91 @@ def execute(self, userdata):
 
 
 # TODO
-class Revisit(Objective):
+class Retrace(Objective):
     outcomes = ['found','not_found']
 
+    #TODO: confirm id names
+    target_class_ids = ["dropper", "wolf", "bat"]
+
+
     def __init__(self, task_name, listener):
-        super(Revisit, self).__init__(self.outcomes, task_name + "/Revisit")
\ No newline at end of file
+        super(Retrace, self).__init__(self.outcomes, task_name + "/Retrace")
+        self.listener = listener
+        #TODO: add param
+        self.retrace_hit_cnt = rospy.get_param("tasks/dropper/retrace_hit_count")
```

Comment:
We should try and use self.name (or task_name) for params, (or lower(self.name)) in this case, so if we ever change it, we only have to change it in one place

Reply:
good idea